mopro-core/Cargo.toml

```diff
@@ -7,12 +7,25 @@ edition = "2021"
 
 [features]
 default = ["wasmer/dylib"]
+halo2 = ["halo2_proofs", "halo2-circuit"]
```

Comment:
Could we be consistent about either using `_` or `-`?

Reply:
I can switch `"halo2-circuit"` to `"halo2_circuit"` as this is a local dependency, as opposed to `"halo2_proofs"`, yet this seems to be a larger issue with rust dependencies using mostly `"-"` than `"_"`, hence this will stand out.  For example in the rest of the dependencies for circom, we also have both: `"once_cell"` and `"ark-ec"`. 

I propose to keep this until we rework the file to separate Circom dependencies from shared dependencies.